Tools/Taxonomy/Common/Options.cs

```diff
@@ -0,0 +1,78 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using CommandLine;
+
+namespace Taxonomy
+{
+    internal interface IOptions
+    {
+        [Value(0, MetaName = "source file path", HelpText = "Source file path to process.", Required = true)]
+        string SourceFilePath { get; set; }
+
+        [Value(1, MetaName = "target file path", HelpText = "Target file path to save.", Required = true)]
```

Comment:
>[Value(1, MetaName = "target file path", HelpText = "Target file path to save.", Required = true)] [](http://example.com/codeflow?start=8&length=98)

you could use:
[Value("output-file-path", helptext, required)]

If you apply to values 1,2,3. You could use in any order.

Reply:
Done and tried the sarif files generated the same.